lib/debugger.coffee

```diff
@@ -31,11 +35,15 @@ class ProcessManager extends EventEmitter
         appArgs = @atom.config.get('node-debugger.appArgs')
         port = @atom.config.get('node-debugger.debugPort')
         env = @parseEnv @atom.config.get('node-debugger.env')
+        defaultFile = @atom.project.getPaths()[0] + '/' + @atom.config.get('node-debugger.defaultFile')
```

Comment:
atom.project.getPaths() can return zero or more paths it seems like. When trying it out a bit this was my conclusions:
If the file menu option "Open folder" is used the result will contains one single path.
In atom you can use the file menu option "Add Project Folder..." to add multiple paths.
If none is used it might be empty.